sqld/src/namespace.rs

```diff
@@ -381,7 +381,11 @@ pub type DumpStream =
 
 fn make_bottomless_options(options: &Options, name: &Bytes) -> Options {
     let mut options = options.clone();
-    let db_id = format!("ns-{}", std::str::from_utf8(name).unwrap());
+    let db_id = format!(
+        "ns-{}:{}",
+        std::str::from_utf8(name).unwrap(),
+        options.db_id.unwrap_or_default()
```

Comment:
this way we can prefix-search all namespaces in a scope
```suggestion
        options.db_id.unwrap_or_default()
        std::str::from_utf8(name).unwrap(),
```

Reply:
Changed